--- a/libtiff/tif_win32.c
+++ b/libtiff/tif_win32.c
@@ -226,6 +226,7 @@ TIFF*
 TIFFFdOpen(int ifd, const char* name, const char* mode)
 {
        TIFF* tif;
+       thandle_t clientdata;
        int fSuppressMap;
        int m;
        fSuppressMap=0;
@@ -237,7 +238,10 @@ TIFFFdOpen(int ifd, const char* name, const char* mode)
                        break;
                }
        }
-       tif = TIFFClientOpen(name, mode, (thandle_t)ifd, /* FIXME: WIN64 cast to pointer warning */
+
+       clientdata = (thandle_t)_get_osfhandle(ifd);
+
+       tif = TIFFClientOpen(name, mode, clientdata,
                        _tiffReadProc, _tiffWriteProc,
                        _tiffSeekProc, _tiffCloseProc, _tiffSizeProc,
                        fSuppressMap ? _tiffDummyMapProc : _tiffMapProc,
