diff --git a/thumbnailer/meson.build b/thumbnailer/meson.build
index e801144..8d4c7a6 100644
--- a/thumbnailer/meson.build
+++ b/thumbnailer/meson.build
@@ -5,7 +5,8 @@ executable('gdk-pixbuf-thumbnailer',
              '-DTHUMBNAILER_USAGE="Thumbnail images"',
            ],
            dependencies: gdk_pixbuf_deps + [ gdkpixbuf_dep ],
-           install: true)
+           install: true,
+           install_rpath: gdk_pixbuf_libdir)
 
 gdk_pixbuf_print_mime_types = executable('gdk-pixbuf-print-mime-types',
                                          'gdk-pixbuf-print-mime-types.c',
diff --git a/gdk-pixbuf/meson.build b/gdk-pixbuf/meson.build
index 7c58bef..6c32046 100644
--- a/gdk-pixbuf/meson.build
+++ b/gdk-pixbuf/meson.build
@@ -179,7 +179,8 @@ gdkpixbuf = library('gdk_pixbuf-2.0',
                       gdk_pixbuf_deps,
                       included_loaders_deps,
                     ],
-                    install: true)
+                    install: true,
+                    install_rpath: gdk_pixbuf_libdir)
 
 build_gir = get_option('gir')
 if build_gir
@@ -303,7 +305,8 @@ foreach bin: gdkpixbuf_bin
                    dependencies: gdk_pixbuf_deps + [ gdkpixbuf_dep ],
                    include_directories: [ root_inc, gdk_pixbuf_inc ],
                    c_args: common_cflags + gdk_pixbuf_cflags,
-                   install: true)
+                   install: true,
+                   install_rpath: gdk_pixbuf_libdir)
 
   # Used in tests
   set_variable(bin_name.underscorify(), bin)
