diff --git a/src/google/protobuf/parse_context.h b/src/google/protobuf/parse_context.h
index c0427bc..8635314 100644
--- a/src/google/protobuf/parse_context.h
+++ b/src/google/protobuf/parse_context.h
@@ -744,7 +744,6 @@ PROTOBUF_EXPORT PROTOBUF_MUST_USE_RESULT const char* PackedEnumParser(
     void* object, const char* ptr, ParseContext* ctx);
 
 template <typename T>
-PROTOBUF_EXPORT_TEMPLATE_DEFINE
 PROTOBUF_MUST_USE_RESULT const
     char* PackedEnumParser(void* object, const char* ptr, ParseContext* ctx,
                            bool (*is_valid)(int), InternalMetadata* metadata,
@@ -760,7 +759,6 @@ PROTOBUF_MUST_USE_RESULT const
 }
 
 template <typename T>
-PROTOBUF_EXPORT_TEMPLATE_DEFINE
 PROTOBUF_MUST_USE_RESULT const
     char* PackedEnumParserArg(void* object, const char* ptr, ParseContext* ctx,
                               bool (*is_valid)(const void*, int),
